userguide/tn_vcenter.rst

```diff
@@ -4,747 +4,12 @@
 vCenter Plugin
 ==============
 
-
-The %brand% vCenter Plugin integrates the control and operation of
+The %brand% vCenter plugin integrates the control and operation of
```

Comment:
s/plugin/Plugin/

This should go into more depth about the plugin capabilities and why the user needs it.